app/controllers/show_assign_value.rb

```diff
@@ -0,0 +1,20 @@
+class ShowAssignValue
+  def set_value(resume_game)
+    player = resume_game.player
+    @opponent = resume_game.opponent
+    current_player = resume_game.current_player
+    board = resume_game.board
+    toggle = TicTacToeGame::Toggle.new(player)
+    game = TicTacToeGame::Game.new(board, player, toggle)
+    winner = get_toggle.current_turn(current_player)
+    game_over = GameoverType.new
+    game_over.message(game, winner)
+  end
+
+  def get_toggle
+    if @opponent == "human"
```

Comment:
I find the code is more readable when you simply pass in `opponent` rather than assigning it as an instance variable outside of the function. Because it is easier to trace back to where the variable came from.